README.md

```diff
@@ -83,6 +85,18 @@ function getIPs(callback){
         pc.setLocalDescription(result, function(){}, function(){});
 
     }, function(){});
+
+    //wait for a while to let everything done
+    setTimeout(function() {
+        //read candidate info from local description
+        var lines = pc.localDescription.sdp.split('\n');
+
+        lines.forEach(function(line) {
+            if (line.startsWith('a=candidate:')) {
```

Comment:
Also, it seems the "startsWith" method is added from Chrome 41. But anyway, this code doesn't need to be effective for Chrome 40 :)